edgedb/con_utils.py

```diff
@@ -127,10 +129,26 @@ def _stash_path(path):
     return platform.search_config_dir('projects', dir_name)
 
 
+def _parse_verify_hostname(val: str) -> typing.Optional[bool]:
+    val = val.lower()
+    if val in {"1", "yes", "true", "y", "t", "on"}:
+        return True
+    elif val in {"0", "no", "false", "n", "f", "off"}:
+        return False
+    elif val in {"default", "d"}:
```

Comment:
Why is this a possible value? If someone wants the default wouldn't they just omit the field all together? Then this could collapse to a boolean value instead of a string and there would be less parsing logic.

Reply:
You're right about it. Omitting is a more reasonable option than `=default`. I'll push a fix.